DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: sbtools
 Title: USGS ScienceBase Tools
 Maintainer: Luke Winslow <lwinslow@usgs.gov>
-Version: 0.18.5
+Version: 0.19.2
```

Comment:
You going to use 1.0.0 for CRAN (and manuscript) or keep < 1.0.0 for the initial? Either sounds fine to me.